README.md

```diff
@@ -1,2 +1,61 @@
-# corporate-theme
-StoreConnect Corporate Theme
+# Corporate Theme
+
+This is a multi-purpose theme, ideal for corporate, B2B, or services companies.
+
+## Directory Structure
+
+This theme has a specific folder structure. Some folders may be empty, but they need to remain part of the theme structure as they are not overridden or used for new templates.
+
+Also, it has been developed using the [Base Theme](https://github.com/GetStoreConnect/base-theme) as a boilerplate. The Corporate theme doesn't contain all the base theme structure, instead, we only include those customised, modified and new templates, snippets, variables or translations. Feel free to create a brand new theme using the base theme.
+
+## Theme Installer
+
+The theme installer is in your Salesforce org, where you must upload this theme as a zip file. Before uploading, remove the .git directory and README.md file from this repo. The theme installer displays errors if it does not find the standard directory structure.
```

Comment:
Don't think you need to worry about the extra files anymore

Reply:
@toddweetman I'm not fully confident to say that you can upload any type of files as I haven't tested various situations yet.